src/utils.js

```diff
@@ -113,19 +129,48 @@ export function getCSS(
  * - numbers
  *   - `margin={1}` becomes `[1]`
  */
-export function parseSpacing(spacing: any): number[] | void {
-  if (spacing instanceof Array) {
-    return spacing.map(parseFloat)
-  } else if (typeof spacing === 'undefined') {
+
+export function parseSpacing(spacing: any): Array<number> | void {
```

Comment:
This method is now doing a lot. What if, before getting here we just replace all aliases with their values? That way we have to do a lot less changes here

This we are also loosing the `base` meaning which we should keep. alias * base should still work